### PR TITLE
Add repo GitHub link to the project breakout page 

### PR DESCRIPTION
**Pull request by Richie78321**
Add a repo GitHub link to the project breakout page.

* Add support for getting the HTML URL on the backend.
* Add the repo link to a clickable [FontAwesome GitHub icon](https://fontawesome.com/icons/github-square?style=brands) in the project title card.
* Add testing for the presence of a GitHub HTML URL on the project breakout page.

### CODE REVIEW

File: src/main/java/com/google/opensesame/projects/ProjectData.java [hunk 1]
```diff
@@ -158,4 +159,17 @@ public String getRepositoryId() {
 
     return Collections.unmodifiableList(mentors);
   }
+
+  /**
+   * Gets the URL of the GitHub page for the repository.
+   *
+   * @return Returns the URL of the GitHub page.
+   */
+  public String getGitHubHtmlUrl() {
```
Comment (billyeh) [inline src/main/java/com/google/opensesame/projects/ProjectData.java:168]: Just wondering why we have Html in there? Doesn't seem to have to do with Html to me, but I might be misunderstanding something.

File: src/main/webapp/js/tests/ExpandedProject.test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {ExpandedProject} from '../projects/react/components/ExpandedProject.js';
+import {ExpandedProject, PROJECT_GITHUB_ICON_TITLE} from '../projects/react/components/ExpandedProject.js'; // eslint-disable-line
```
[the remark this answers is not in the document]
Comment (Richie78321) [reply src/main/webapp/js/tests/ExpandedProject.test.js:2]: Yup, [there sure is](https://google.github.io/styleguide/jsguide.html#es-module-imports)! 

> Import statements must not be line wrapped and are therefore an exception to the 80-column limit.

I was pretty sad about it as well.

File: src/main/java/com/google/opensesame/projects/ProjectData.java (hunk 1 above)
Comment (Richie78321) [reply src/main/java/com/google/opensesame/projects/ProjectData.java:168]: You make a good point. I thought it'd be best to match the naming used in the Java GitHub API, but on second thought I think I'll change it, as it doesn't seem necessary. Thanks!

File: src/main/webapp/js/tests/ExpandedProject.test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (billyeh) [reply src/main/webapp/js/tests/ExpandedProject.test.js:2]: Thanks for the link! nit: can add the eslint error code after eslint-disable-line to make it specific/obvious which error we're suppressing.

**Pull request merged**